packages/connect/src/data/connectSettings.ts

```diff
@@ -25,7 +25,9 @@ const initialSettings: ConnectSettings = {
     popup: true,
     popupSrc: `${DEFAULT_DOMAIN}popup.html`,
     webusbSrc: `${DEFAULT_DOMAIN}webusb.html`,
+    // deprecated, use transports instead
```

Comment:
Is this comment a `TODO`? I do not totally understand the reason of it.

Reply:
it is a note. in connect, we still want to stay backwards compatible so we still keep webusb field here. but once we lets say release next major version of connect, we can remove all of these. It would be nice to create a convention how to mark places that can be changed after releasing next major version. any ideas?

Reply:
I think using `TODO` with a notation makes sense and I have seen it in other places like `TODO(major-version): .. .....`.

That helps you then to find all occurrences of it in the code base.

Reply:
maybe there is an eslint rule that will enforce that all comments need to have some prefix
```todo: something```
```note: something```
```breaking: something``` 
and then we could find stuff by those prefixes